packages/react-native-lockdown/test/unit/assertPolyfills.spec.js

```diff
@@ -1,4 +1,10 @@
-const test = require('ava')
+'use strict'
+
+/**
+ * @import {TestFn} from 'ava'
+ */
+
+const test = /** @type {TestFn} */ (/** @type {unknown} */ (require('ava')))
```

Comment:
I'm not sure why AVA's CJS types are so totally broken, even in the latest version.